vendor/assets/stylesheets/fenrir/building_blocks/mixins/_wrappers.scss

```diff
@@ -0,0 +1,126 @@
+
+
+
+//----------------------------------------------\\
+// Charlie slice wrapper
+//----------------------------------------------\\
+
+@mixin w-slice() {
+  padding-left: 15px;
+  padding-right: 15px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+
+  &--no-wrap {
+    flex-wrap: nowrap;
+  }
+
+  &--no-padding {
+    padding: 0;
+  }
+}
+
+
+
+//----------------------------------------------\\
+// Charlie card wrapper
+//----------------------------------------------\\
+
+@mixin w-card() {
+  position: relative;
+  width: 100%;
+  border-radius: $br;
+  margin: 15px 0;
+  box-shadow: 0 0 0 1px color('black', A5);
```

Comment:
Not necessarily, but it can be overridden on a card by card basis, so this is more of a default. I.e. a Modal card might have:
```
@include w-card;
box-shadow: 0 2px 4px 0 color('black', A25);
```

There's also the case of some cards, i.e. our call to action cards actually being moved to using the button wrapper instead of the card wrapper to make them look clickable.